ci-operator/step-registry/operator/hypershift/operator-hypershift-workflow.yaml

```diff
@@ -13,13 +13,15 @@ workflow:
       ZONES_COUNT: "1"
       PARALLEL: "true"
       FIREWATCH_JIRA_SERVER: https://issues.redhat.com
+      DISABLE_WORKLOAD_MONITORING: "false"
+      ETCD_ENCRYPTION: "false"
+      FIPS: "false"
     post:
-    - ref: operator-uninstall
     - chain: rosa-aws-sts-deprovision-byo-vpc
     - ref : firewatch-report-issues
     pre:
     - chain: rosa-aws-sts-provision-byo-vpc
-    - ref: osd-ccs-conf-idp-htpasswd-multi-users
+    # - ref: osd-ccs-conf-idp-htpasswd-multi-users
```

Comment:
meant to be removed ?

Reply:
for now no; once we have all flows working will remove